content/tutorials/earthworks/levees.qmd

```diff
@@ -0,0 +1,616 @@
+m.show()
+m.save("images/levees_02.webp")
+```
+
+:::
+
+![Terrain](images/levees_02.webp)
+
+## Flood Simulation
+
+Use {{< meta links.r_lake >}} to model innudation from storm surge. 
+Set the water level to one meter and the coordinates to `1109000, 102750`.
+The simulation will show that the degraded ridge
+would be overtopped by a meter of surge.
+
+::: {.panel-tabset group="language"}
+
+## Command line
+
+```{bash}
+r.lake elevation=elevation water_level=1 lake=flooding coordinates=1109000,102750
+d.shade shade=relief color=elevation brighten=36
+d.rast map=flooding
+d.legend raster=flooding digits=1 color=white at=5,95,1,3
+```
+
+## Python
+
+```{python}
+# Simulate flooding
+gs.run_command(
+    "r.lake",
+    elevation="elevation",
+    water_level=1,
+    lake="flooding",
+    coordinates=[1109000, 102750]
+    )
+
+# Display flooding
+m = gj.Map(width=800)
+m.d_shade(shade="relief", color="elevation", brighten=36)
+m.d_rast(map="flooding")
+m.d_legend(raster="flooding", digits=1, color="white", at=(5, 95, 1, 3))
+m.show()
+m.save("images/levees_03.webp")
+```
+
+:::
+
+![Flood simulation](images/levees_03.webp)
+
+## Coastal Restoration
+
+Map the proposed coastal infrastructure projects. 
+Layer the vector maps of proposed ridge restoration, 
+levee improvement, and gap closure projects
+over the shaded relief. 
+
+::: {.panel-tabset group="language"}
+
+## Command line
+
+```{bash}
+d.shade shade=relief color=elevation brighten=36
+d.vect map=levees color=white width=4
+d.vect map=ridges color=yellow width=4
+d.vect map=gaps icon=basic/pin_dot size=25 color=white fill_color=white
+d.text text="Levee improvement" color=white size=7 at=22,10
+d.text text="Ridge restoration" color=yellow size=7 at=34,65
+d.text text="Gap closure" color=white size=7 at=49,10
+d.legend raster=flooding digits=1 color=white at=5,95,1,3
+```
+
+## Python
+
+```{python}
+# Display projects
+m = gj.Map(width=800)
+m.d_shade(shade="relief", color="elevation", brighten=36)
+m.d_vect(map="levees", color="white", width=4)
+m.d_vect(map="ridges", color="yellow", width=4)
+m.d_vect(map="gaps", icon="basic/pin_dot", size=25, color="white", fill_color="white")
+m.d_text(text="Levee improvement", color="white", size=7, at=(22,10))
+m.d_text(text="Ridge restoration", color="yellow", size=7, at=(34,65))
+m.d_text(text="Gap closure", color="white", size=7, at=(49,10))
+m.show()
+m.save("images/levees_04.webp")
+```
+
+:::
+
+![Coastal restoration and protection projects](images/levees_04.webp)
+
+# Ridge Restoration
+
+Use {{< meta links.r_earthworks >}}
+to model the restored ridge and 
+the borrow needed to build it.
+The restored ridge should have
+a crest elevation of 1.5 meters,
+a crest width of 15.2 meters,
+and 20% slopes with a ratio of 5H:1V.
+The soil to rebuild the ridge
+should be taken from a borrow area
+that would be offset by 7.6 meters 
+from the foot of the ridge. 
+
+## Borrow Area
+
+Model the borrow area
```

Comment:
Please explain the terminology, borrow is not a commonly used word. There may be others like that.